### PR TITLE
@types/node-fetch: Remove a hyphen that broke a import

### PR DESCRIPTION
**Pull request by nicolas377**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

There was a hyphen in form-data (the actual package is called formdata now.

### CODE REVIEW

File: types/node-fetch/index.d.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 /// <reference types="node" />
 
-import FormData = require('form-data');
```
Comment (nicolas377) [inline types/node-fetch/index.d.ts:18]: Throws "Cannot find module 'form-data' or its corresponding type declarations. ts(2307)"

Comment (OmgImAlexis) [reply types/node-fetch/index.d.ts:18]: https://www.npmjs.com/package/form-data

**Comment by OmgImAlexis** [comment]
The package doesn’t seem to have changed names. https://www.npmjs.com/package/form-data

**Comment by nicolas377** [comment]
node-fetch depends on @types/formdata, which got renamed from form-data
the package itself is what @types/formdata depends on, but not @types/node-fetch

**Comment by peterblazejewicz** [comment]
Original package depends on `form-data`:
https://github.com/node-fetch/node-fetch/blob/master/package.json#L57

**Comment by nicolas377** [comment]
I actually didn't see that! Thanks! PR still applies

**Comment by nicolas377** [comment]
oh wait, it doesnt

**Comment by nicolas377** [comment]
If you can, can you review this pr? #54823

**Pull request closed without merging**